app/templates/events/import.hbs

```diff
@@ -1 +1,2 @@
-import data
+{{events/event-import-section}}
```

Comment:
If I'm not wrong `{{outlet}}` is used to render the nested routes. `Import` is nested under `events`, we must have `{{outlet}}` to render the sub-routes of `events`. Are you suggesting to add sub-routes to `import`?